### PR TITLE
Add Pull Request templates

### PR DESCRIPTION
**Pull request by danielfdsilva**
Create a series of PR templates to help with content checks.
This is a start and we can improve them and add tasks as we find more pain points.

It will only be possible to see them in action once this is merged.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/content.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+## Content checklist
+
+Check the [Content Documentation](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/CONTENT.md) for more details.
+
+### Any content
+- [ ] Cover image with proper attribution (when relevant) is present.
+- [ ] Cover images do not contain any text (they're used as a background and text may not be readable).
+- [ ] All images are not to big/heavy. As a general rule of thumb they should not weigh more than 500KB, or be larger than 2000px. See the [media guide](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/frontmatter/media.md#media) for more info.
```
Comment (nerik) [inline .github/PULL_REQUEST_TEMPLATE/content.md:14]:
```suggestion
- [ ] All images are not too big/heavy. As a general rule of thumb, they should not weigh more than 500KB, or be larger than 2000px. See the [media guide](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/frontmatter/media.md#media) for more info.
```

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## What am I changing and why
+
+_Replace with brief description of the update and why it is needed. Link to relevant issues._
+
+## How to test
+_Replace with instructions on ow this change be tested/verified._
```
Comment (nerik) [inline .github/PULL_REQUEST_TEMPLATE/default.md:6]:
```suggestion
_Replace with instructions on how this change be tested/verified._
```

**Review by nerik — APPROVED** [review]
Awesome 👍 
Not sure but I think we discussed automated checks for images at some point. Is that something worth looking at?

**Comment by danielfdsilva** [comment]
@nerik This is a quick-win approach as suggested by @hanbyul-here . If this does not bear fruit we can deploy more drastic measures :p

**Pull request merged**